commands/JetpackSiteModuleSetStatus.php

```diff
@@ -0,0 +1,136 @@
+<?php
+
+namespace WPCOMSpecialProjects\CLI\Command;
+
+use Symfony\Component\Console\Attribute\AsCommand;
+use Symfony\Component\Console\Command\Command;
+use Symfony\Component\Console\Input\InputArgument;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Output\OutputInterface;
+use Symfony\Component\Console\Question\Question;
+
+/**
+ * Set the on/off status for a given Jetpack module on a given site.
+ */
+#[AsCommand( name: 'jetpack:set-site-module-status' )]
```

Comment:
What do you think of using here `jetpack:toggle-module` (or `jetpack:toggle-site-module-status`)? The former is easier to remember

Reply:
That makes sense. I think using aliases is a great idea.